### ISSUE TITLE
When pressing "alt" to show the menu bar, keyboard focus is not given to it

### ISSUE THREAD

**Issue by MrTrillian** [Microsoft/vscode#2945]
Repro:
- Use "View/Toggle Menu Bar" to hide the menu bar
- Press and release "alt", the menu bar appears
- Try using the arrow keys or mnemonic keys ('F' for 'File' menu)

Actual: there is no way to interact with the menu bar from the keyboard
Expected: same as the Hide Menu Bar extension for VS, where pressing alt gives keyboard focus to the "file menu", which allows navigating with the keyboard arrows or mnemonic keys.

Windows 10, VSCode 0.10.8

**Comment by stkb** [comment]
To add some information, if you hold ```alt``` and press ```f```, then it works as expected, but not if you press them separately.

**Comment by Tyriar** [comment]
It looks like this is just how [Electron](https://github.com/atom/electron) behaves under Windows 10, you may want to file an issue there.

**Comment by bpasero** [comment]
Might be related to #1569

**Comment by benliddicott** [comment]
Definitely related to https://github.com/Microsoft/vscode/issues/1569

**Comment by benliddicott** [comment]
Appears to be a duplicate of https://github.com/Microsoft/vscode/issues/1569

* See also https://github.com/Microsoft/vscode/issues/45186 
* See also https://github.com/Microsoft/vscode/issues/36077

**Comment by wolf99** [comment]
AFAICT the pressing and holding trick mentioned by @stkb  doesn't work (or has stopped working) as this bug randomly appeared fro me today - everything worked as normally expected previously

**Comment by sbatten** [comment]
This is resolved in our custom menubar implementation via #53924. The new menubar can be enabled via the setting 'window.titlebarStyle' => `custom`
